modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -299,16 +301,12 @@ public RocksDbKeyValueStorage(
         );
 
         this.dbPath = dbPath;
+        this.scheduledExecutor = scheduledExecutor;
```

Comment:
Is this executor used for scheduling absolutely any tasks, or for some specific tasks (like handling RocksDB-related stuff)? If the latter is true, would it make sense to rename it to reflect its purpose?

Reply:
I'll add a description.